### PR TITLE
cUNI mainnet

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

File: networks/mainnet.json [hunk 1]
```diff
@@ -28,26 +27,30 @@
...
         "Base200bps_Slope222bps_Kink90_Jump40": "0x5562024784cc914069d67D89a28e3201bF7b57E7",
         "SAI": "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359",
-        "StdComptrollerG4": "0xAf601CbFF871d0BE62D18F79C31e387c76fa0374",
         "REP": "0x1985365e9f78359a9B6AD760e32412f4a445E862",
         "cZRX": "0xB3319f5D18Bc0D84dD1b4825Dcde5d5f7266d407",
         "cWBTC": "0xC11b1268C1A384e55C48c2391d8d480264A3A7F4",
         "USDC": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
         "PriceData": "0xc629c26dced4277419cde234012f8160a0278a79",
-        "PriceFeed": "0x9b8eb8b3d6e2e0db36f41455185fef7049a35cae",
+        "PriceFeed": "0x922018674c12a7f0d394ebeef9b58f186cde13c1",
```
Comment (coburncoburn) [inline networks/mainnet.json:53]: May want to keep old price feed in here... api will need it to sync at least

File: networks/mainnet.json [hunk 2]
```diff
@@ -61,7 +64,10 @@
         "Pair_BAT_ETH": "0xb6909b960dbbe7392d405429eb2b3649752b4838",
         "Pair_ETH_USDC": "0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc",
         "PriceFeedPoster": "0x83563ba7f1b093aae57fe876f8d870f8a1508886",
-        "CrowdProposalFactory": "0xB5212a2fa63c1863b9e8670e2A6D420d0309c502"
+        "CrowdProposalFactory": "0xB5212a2fa63c1863b9e8670e2A6D420d0309c502",
+        "StdComptrollerG5": "0x7b5e3521a049C8fF88e6349f33044c6Cc33c113c",
```
Comment (coburncoburn) [inline networks/mainnet.json:68]: this in here 2x?

File: networks/mainnet.json [hunk 3]
```diff
@@ -87,22 +93,28 @@
...
         "Base200bps_Slope222bps_Kink90_Jump40": 8983555,
-        "StdComptrollerG4": 10348750,
         "cZRX": 7710733,
         "cWBTC": 8163813,
         "PriceData": 10551018,
-        "PriceFeed": 10551058,
-        "CrowdProposalFactory": 10829893
+        "PriceFeed": 10921522,
+        "CrowdProposalFactory": 10829893,
+        "StdComptrollerG5": 10809458,
```
Comment (coburncoburn) [inline networks/mainnet.json:115]: double bubble

File: networks/mainnet.json [hunk 4]
```diff
@@ -529,16 +492,43 @@
...
+          "description": "DAIInterestRateModelV3 jump=1090000000000000000 kink=800000000000000000 pot=0x197e90f9fad81970ba7976f33cbd77088e5d7cf7 jug=0x19c0976f590d67707e62397c87829d896dc0f1f1 owner=0x6d903f6003cca6255d85cca4d3b5e5146dc33925",
+          "jump": "1090000000000000000",
           "kink": "800000000000000000",
           "pot": "0x197e90f9fad81970ba7976f33cbd77088e5d7cf7",
           "jug": "0x19c0976f590d67707e62397c87829d896dc0f1f1",
+          "owner": "0x6d903f6003cca6255d85cca4d3b5e5146dc33925",
           "address": "0xfeD941d39905B23D6FAf02C8301d40bD4834E27F"
         },
+
+        "IRM_USDT_Updateable": {
```
Comment (coburncoburn) [inline networks/mainnet.json:508]: glad thes irms are in here now

File: networks/mainnet.json (hunk 2 above)
Comment (jflatow) [reply networks/mainnet.json:68]: No those are remove/add fixing the missing comma

Comment (coburncoburn) [reply networks/mainnet.json:68]: StdComptrollerG5 is on line 45 & 67 both

Comment (jflatow) [reply networks/mainnet.json:68]: 🎣

**Review by coburncoburn — APPROVED** [review]

File: networks/mainnet.json [hunk 5]
```diff
@@ -498,8 +455,18 @@
...
             "initial_exchange_rate_mantissa": "20000000000000000",
             "address": "0xC11b1268C1A384e55C48c2391d8d480264A3A7F4"
+        },
+        "cUNI": {
+            "name": "Compound Uniswap",
+            "symbol": "cUNI",
+            "decimals": 8,
+            "underlying": "0x1f9840a85d5af5bf1d1762f925bdaddc4201f984",
+            "contract": "CErc20Delegator",
+            "intial_exchange_rate_mantissa": "200000000000000000000000000",
```
Comment (coburncoburn) [inline networks/mainnet.json:465]: typo "intial" -> "initial"

**Pull request merged**